ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParser.scala

```diff
@@ -31,6 +35,19 @@ object WorkflowConfigParser {
   case class ParserError(details: String)
 
   object Decoders {
+    implicit val scalaDurationDecoder: Decoder[FiniteDuration] =
```

Comment:
there must be a default circe decoder to support FiniteDurations

Reply:
If there is I can't find it.

What I did find though is a duration decoder but for typesafe config, not yaml: https://circe.github.io/circe-config/io/circe/config/syntax$.html

Reply:
yeah, i could not find it either, so just ignore my comment then :)